packages/elements-core/src/components/TryIt/RequestBody.tsx

```diff
@@ -43,3 +26,34 @@ export const RequestBody: React.FC<RequestBodyProps> = ({ examples, requestBody,
     </Panel>
   );
 };
+
+function ExampleMenu({ examples, requestBody, onChange }: RequestBodyProps) {
+  const handleClick = React.useCallback(
+    (example: INodeExample | INodeExternalExample) => {
+      onChange(safeStringify('value' in example ? example.value : example.externalValue, undefined, 2) ?? requestBody);
```

Comment:
Do we ever want externalValue in there? Unless http-spec is doing something special, externalValye is a URI to a file. We'd need to follow the URI, see if we understand the file, and return the data (which is already going to be JSON or similar). 

Or we can ignore externalValue for examples, but we cannot use them interchangeably with value.

Reply:
I didn't change the logic of this code in this PR - just updated for the new Menu interface. If ya'll think this is important I can do it now or you guys could tackle in separate PR - cc @marcelltoth @mpodlasin